lib/sign-up.js

```diff
@@ -23,6 +23,8 @@ function signUp (state, options) {
     return Promise.reject(new Error('SignUp with profile data not yet implemented. Please see https://github.com/hoodiehq/hoodie-account-client/issues/11.'))
   }
 
+  options.createdAt = new Date().toISOString()
```

Comment:
We should set `createdAt` to the timestamp when the user loads a page for the first time. The idea is that we want to know the timestamp when a user open a website of a Hoodie app for the first time, so we can compare the `createdAt` to the `signedUp` (will be set on server) timestamps for analytics.

I would suggest to check if `createdAt` is set here: https://github.com/hoodiehq/hoodie-account-client/blob/master/utils/get-state.js#L27 and if it’s not, set it directly on the `storedAccount` variable. 

Does that make sense?